lnbits/extensions/gerty/views_api.py

```diff
@@ -93,6 +93,13 @@ async def api_gerty_satoshi():
     return await get_satoshi()
 
 
+@gerty_ext.get("/api/v1/gerty/pages/{gerty_id}")
```

Comment:
- is this just a place-holder for future logic to be added?

Reply:
Looks like it. Removed for now.